app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -70,6 +70,11 @@ angular.module('Bastion.components').factory('Nutupane',
                 replace = replace || false;
                 table.working = true;
 
+                if (table.initialLoad) {
+                    table.refreshing = true;
+                    table.initialLoad = false;
```

Comment:
would it make sense to check if the current resource count is 0 rather than having a 2nd variable?